### DISCUSSION TITLE
Code signing and ESET / McAfee.

### DISCUSSION THREAD

**Discussion by PC-Support-Services** [amidaware/tacticalrmm#2035] [Q&A]
Will paying for code signing allow machines with McAfee and ESET to trust the software and install it rather than getting caught and auto-deleted?

**Comment by silversword411** [comment]
https://docs.tacticalrmm.com/av/#antivirus
https://docs.tacticalrmm.com/code_signing/#tactical-rmm-is-getting-flagged-as-pua-or-virus-will-this-fix-it
